assets/js/modules/analytics/components/settings/SettingsEdit.js

```diff
@@ -54,7 +58,11 @@ export default function SettingsEdit() {
 	let viewComponent;
 	// Here we also check for `hasResolvedAccounts` to prevent showing a different case below
 	// when the component initially loads and has yet to start fetching accounts.
-	if ( isDoingSubmitChanges || ! hasResolvedAccounts ) {
+	if (
+		isDoingSubmitChanges ||
+		! hasResolvedAccounts ||
+		hasModuleAccess === undefined
```

Comment:
We also need to consider module access checks could result in a request error which would leave this undefined and in an infinite loading state which means we'll need to rely on `hasFinishedResolution` like the others, only we'll need to "calculate" which resolvers need checking (if at all) because these won't run for the owner (but also with ga4 or not).